source/mapping-rules/fields/field-customizations.txt

```diff
@@ -0,0 +1,102 @@
+.. _rm-field-customizations:
+
+======
+Fields
+======
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can customize fields when adding or editing your mapping rules. 
+Relational Migrator allows you to define custom fields using 
+:ref:`rm-calculated-fields`, define custom IDs using 
+:ref:`rm-id-field-customizations`, or convert your source 
+relational column data types into :ref:`a supported BSON data type<rm-field-supported-types>`.
+
+Use Cases
+---------
+
+You can use field customization to:
+
+- Include or exclude fields in your mapping rules.
+- Change field names.
+- Rename fields to include a dotted path, which results in 
+  that field embedded in a parent document. For details, see 
+  :ref:`embedding fields<rm-embedding-individual-fields>`.
+- Customize ``_id`` fields using JavaScript expressions.
+- Customize a field using JavaScript expressions.
+- Configure null handling behavior.
+
+Behavior
+--------
+
+The fields section contains a list of fields derived from columns 
+in the relational table. Mapping rules include each column by default. 
+You can exclude a field from your sync job by unchecking the box. 
+The field name initially matches the column name (with the 
+:ref:`global casing <rm-global-casing>` applied). Modify the 
+name by editing the text box. You can click the :icon-fa5:`chevron-left`
+icon next to each field to reveal additional options.
+
+- Relational Migrator assigns a default data type to your MongoDB fields 
+  that most closely matches the source column data type. 
+
+- You can use the :guilabel:`BSON Type` drop-down menu to change the MongoDB 
+  field data type to a different data type to fine-tune your schema. 
+  If the chosen type is not compatible with the source values, each 
+  field converted results in an error during the sync job. Sync jobs 
+  continue until the user-defined maximum error threshold is reached.
+
+.. _rm-embedding-individual-fields:
+
+- You can rename a field and include a ``.`` to embed fields in the 
```

Comment:
[suggestion] Consider starting the sentence with the user's goal. Applies globally. 

"To embed child fields in a parent document, rename the field in the form ``parent.child``. For example..."